banzai/logs.py

```diff
@@ -37,5 +37,5 @@ def _image_to_tags(image_config):
 
 
 def set_log_level(log_level='INFO'):
-    for handler in logging.getLogger().handlers:
-        handler.setLevel(log_level.upper())
+    root_logger = logging.getLogger()
+    root_logger.setLevel(log_level.upper())
```

Comment:
Just to note, this change also requires you to set the handler level in `context.py`, but anyway it should all be taken care of by the hotfix/preview_logging branch.

Reply:
Ok. I'll merge in that.

Reply:
Ok. That should be merged.